src/Tasks/Exec.cs

```diff
@@ -54,6 +54,9 @@ public Exec()
         private Encoding _standardOutputEncoding;
         private string _command;
 
+        // '^' before _any_ character escapes that character, don't escape it.
+        private char[] _charactersToEscape = { '(', ')', '=', ';', '!', ',', '&', ' '};
```

Comment:
`private static readonly`. No need to create an array per `Exec` instance as it is immutable.